main.cpp

```diff
@@ -59,85 +59,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: woman
+//  action 1: the woman kicks
+woman.kick();
+//  action 2: the woman speaks
+woman.speaks();
+//  action 3: the woman shouts 
+woman.shouts();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:  team
+//  action 1: the team wins
+team.wins();
+//  action 2: the team loses
+team.lose();
+//  action 3: the team goes
+team.goes();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: landspeeder
+//  action 1: the landspeeder flies
+landspeeder.fly()
+//  action 2: the landspeeder stops
+landspeeder.stop();
+//  action 3: the landspeeder turns
+landspeeder.turn();
```

Comment:
we could argue about whether or not this noun should be `camelCased`, but let's not so you get to experience a Code Review and a Change Request.
- [ ] camelCase the `landspeeder` name.